.releaserc

```diff
@@ -0,0 +1,84 @@
+{
+  "branches": [
+    "main",
+    {
+      "name": "dev",
+      "prerelease": true
+    }
+  ],
+  "plugins": [
+    [
+      "@semantic-release/commit-analyzer",
+      {
+        "releaseRules": [
+          {
+            "type": "build",

```

Comment:
What's the motivation behind this?

Reply:
Bumping patcher dependency (or updating other dependencies) should be bumped as a patch change as it somewhat a feature, but not really.

Reply:
Bumping patcher or any other dependency shouldn't create a release. The issue you are currently trying to solve is that ReVanced Manager gets out of sync with patches if the patcher introduces a breaking change. This issue would be worked around by making build commits release but it remains a workaround on the actual issue that ReVanced Manager can not use the correct patches version it is compatible with due to the api not being able to serve old versions (or branches)

Reply:
Alright, will revert this in a sec.